webpack.config.dev.js

```diff
@@ -31,7 +31,8 @@ export default {
         removeComments: true,
         collapseWhitespace: true
       },
-      inject: true
+      // injection is now handled in the template to allow for absolute paths, fixes issue with nested routes
```

Comment:
I completely understand. I am running this setup with bootstrap but I am using the bootstrap-loader. 

I want to try one another approach on this after sleeping on these changes and seeing a bug in the production build. I will try and find an approach that does not mean taking on injection.